### ISSUE TITLE
keep_alive not working

### ISSUE THREAD

**Issue by leshric** [zacs/ha-dualmodegeneric#31]
I'm getting this in the log:

Logger: homeassistant
Source: custom_components/dualmode_generic/climate.py:814
Integration: dualmode_generic (documentation)
First occurred: 10:42:40 AM (1 occurrences)
Last logged: 10:42:40 AM
Error doing job: Task exception was never retrieved

Traceback (most recent call last):
  File "/config/custom_components/dualmode_generic/climate.py", line 814, in _async_control_heating
    "Keep-alive - Turning off %s", active_entity
UnboundLocalError: local variable 'active_entity' referenced before assignment

active_entity is never set during a call for keep_alive.

**Comment by david-kalbermatten** [comment]
Yes, I'm sorry. This one is me. I forgot to check whether or not `active_entity` was assigned when used here:
```python
_LOGGER.info(
    "Keep-alive - Turning off %s", active_entity
)
```

I moved the assignment further outside in the corresponding function in my PR #32.

**Comment by david-kalbermatten** [comment]
@leshric or @zacs If my PR #32 fixed this issue please feel free to close it ;D

**Comment by zacs** [comment]
Fixed in #32
